images/packages/binaries/lvm2/werf.inc.yaml

```
@@ -0,0 +1,66 @@
+---
+image: {{ $.ImageType }}/{{ $.ImageName }}
+final: false
+fromImage: builder/scratch
+import:
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /out
+  to: /libdevmapper
+  before: setup
+
+---
+{{- $version := "v2_03_32" }}
```

Use this to get the version from a file `component_versions/version_map.yml`
```suggestion
{{- $version := get $.Package $.ImageName }}
```